src/main/java/org/kainos/ea/cli/Job.java

```diff
@@ -0,0 +1,57 @@
+package org.kainos.ea.cli;
+
+public class Job {
+    private int jobID;
+    private String jobRole;
+    private String band;
+    private String specifications;
+    private String responsibilities;
+
+    public Job(int jobID, String jobRole, String band, String specifications, String responsibilities) {
+        this.jobID = jobID;
+        this.jobRole = jobRole;
+        this.band = band;
+        this.specifications = specifications;
+        this.responsibilities = responsibilities;
+    }
+
+    public int getJobID() {
+        return jobID;
+    }
+
+    public void setJobID(int jobID) {
+        this.jobID = jobID;
+    }
+
+    public String getJobRole() {
+        return jobRole;
+    }
+
+    public void setJobRole(String jobRole) {
+        this.jobRole = jobRole;
+    }
+
+    public String getBand() {
+        return band;
+    }
+
+    public void setBand(String band) {
+        this.band = band;
+    }
+
+    public String getSpecifications() {
+        return specifications;
+    }
+
+    public void setSpecifications(String specifications) {
+        this.specifications = specifications;
+    }
+
```

Comment:
For both `Specifications` and `Responsibilities` they are plural, could either of these return an array of values?

Reply:
@glenn-hamilton I have moved away from using the Job class so, this was merged in from Mickeys branch and I have changed to use the JobSpecRequest class which just uses the spec and job name for this ticket then next ticket has the responsibilities which takes a List but that is in a second branch Im working on atm.